pubspec.yaml

```diff
@@ -13,7 +15,7 @@ topics:
  - codegen
 
 environment:
-  sdk: ">=3.0.0 <4.0.0"
+  sdk: ">=3.2.0-114.0.dev <4.0.0"
```

Comment:
What's the reason for the generator (ffigen) to require the new SDK version?

Reply:
Listener blocks are built on `NativeCallable.listener`. While that feature did land in 3.1, there are some important updates to it in 3.2 that we need (specifically `keepIsolateAlive = false`).